### ISSUE TITLE
Multi tenancy uses same xero account for all users

### ISSUE THREAD

**Issue by deanzod** [dcblogdev/laravel-xero#17]
I can't see a way to have different users connect their own xero accounts. If I log in as user A and connect xero account A it works. If I log out of both xero and the app, then login as user B, it still says its connected and the tenant_id is the same so it accesses User A's xero account.

When User A first authorises I am storing the tenant_id in the users table. Is there any way to make it force a new oauth flow if a user doesn't have a valid tenant_id?

```php
 $xeroToken = XeroToken::where('tenant_id', auth()->user()->xero_tenant_id)->first();

            if (!$xeroToken) {            
                return redirect()->route('xeroConnect'); // should do completely new oauth flow
            }else{
                 Xero::setTenantId($xeroToken->id); // should connect and continue on as that tenant
            }
```

```php

           Route::get('xero/connect', function () {
                Xero::connect();
                auth()->user()->update(['xero_tenant_id' => Xero::getTenantId()]);
                return redirect()->route('index');
            })->name('xeroConnect');

```

With the above logic instead of doing as I expect, it just adds the same tenant_id to the xero_tokens table in a new row, therefor the same tenant_id to the users table too.

**Comment by tonypartridge** [comment]
At present, we don't seem to have multi-user support. It shouldn't be too hard at add if you want to have a go and do a PR? the maintainer is pretty active.

**Comment by tonypartridge** [comment]
resolved in #11 #16 #19 #
